devel/meson/Portfile

```diff
@@ -57,10 +56,10 @@ post-destroot {
 }
 
 # the following block avoids requiring users to 'sudo port select python3 python37'
-# doing a file test for ${prefix}/bin/python3 and requiring this 
+# doing a file test for ${prefix}/bin/python3 and requiring this
```

Comment:
We usually try to split whitespace changes, typos etc. into a separate commit, but it's so small that it's not really relevant bike-shedding about.